awx/main/models/schedules.py

```diff
@@ -248,11 +256,39 @@ def update_computed_fields(self):
                 self.dtend = future_rs[-1].astimezone(pytz.utc)
             except IndexError:
                 self.dtend = None
+
+        changed = any(getattr(self, field_name) != starting_values[field_name] for field_name in affects_fields)
+        return changed
+
+    def update_computed_fields(self):
+        changed = self.update_computed_fields_no_save()
+        if not changed:
+            return
         emit_channel_notification('schedules-changed', dict(id=self.id, group_name='schedules'))
+        # Must save self here before calling unified_job_template computed fields
+        # in order for that method to be correct
+        self.rrule = Schedule.coerce_naive_until(self.rrule)
+        # do not update modified time
```

Comment:
This comment says "do not update modified time", but you've got it listed in `update_fields` below?

Reply:
that's how we avoid updating the modified time.

https://github.com/ansible/awx/blob/7a835b8c7261fa05d4a13a3996e29702f1923637/awx/main/models/base.py#L161-L162

I'll add a little more verbosity to the comment to call out that practice

Reply:
Oh, I see.